ports/voidstranger/Void Stranger.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+GAMEDIR="/$directory/ports/voidstranger"
+export LD_LIBRARY_PATH="/usr/lib:/$GAMEDIR/lib:$LD_LIBRARY_PATH"
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+# permissions
+$ESUDO chmod 666 /dev/tty0
+$ESUDO chmod +x "$GAMEDIR/lib/splash"
+$ESUDO chmod +x "$GAMEDIR/gmloadernext"
+$ESUDO chmod +x "$GAMEDIR/game_patching.txt"
+$ESUDO chmod 666 /dev/uinput
+
+cd $GAMEDIR
+
+# log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
```

Comment:
Some devices like the trimui smart pro use ash instead of bash. Ash is a more lightweight shell used in embedded systems and has some differences in syntax. Ash does not support process substitution.

Replace `> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1` with `exec > "$GAMEDIR/log.txt" 2>&1`

Reply:
changed, left the current log gen code in a comment

Reply:
waiting to test on trimui before commit